client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -1,15 +1,18 @@
 import { Gridicon, CircularProgressBar } from '@automattic/components';
+import { useSelect } from '@wordpress/data';
 import { useRef, useState } from '@wordpress/element';
 import { Icon, copy } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
 import { useSelector } from 'react-redux';
+import { OnboardSelect } from 'calypso/../packages/data-stores/src';
```

Comment:
[nit] Could we update the import path since it's weird 😓
```suggestion
import type { OnboardSelect } from '@automattic/data-stores';
```